index.html

```diff
@@ -13,44 +13,50 @@
     <title>Thrive Launcher</title>
   </head>
   <body>
-    
+
+    <span class="BottomFade"
+      style="height: 5%; width: calc(35% + 5px); bottom: calc(70px + 22px); left: 8px;">
+    </span>
+
     <span class="ContentBlock"
-          style="height: 90%; width: 40%; top: 0px; left: 0px; border-right: 1px solid 
+          style="height: calc(80% + 10px); width: calc(35% + 5px); top: 10px; left: 8px;
                  rgba(68, 68, 68, 0.46)">
 
-      <h2>Development Forums</h2>
+      <h2>DEVELOPMENT FORUMS</h2>
```

Comment:
What's with the capitalisation change here?

Reply:
I though that will make the box title to be more readable.

Reply:
I don't particularly mind either way. When I posted a test build for other developers no one else seemed to mind either, so... I think this change can stay.